gprofiler/main.py

```diff
@@ -230,8 +239,23 @@ def _strip_container_data(collapsed_data):
             lines.append(line[line.find(';') + 1 :])
         return '\n'.join(lines)
 
+    def send_metadata(self):
```

Comment:
This may raise - wherever you end up putting it, make sure it doesn't stop gProfiler from running

Reply:
Removed in #134